dist/tools/pyterm/pyterm

```diff
@@ -24,8 +24,19 @@ try:
 except ImportError:
     import ConfigParser as configparser
 
-import cmd, serial, socket, sys, threading, readline, time, logging, \
-    os, argparse, re, codecs, signal
+import cmd
+import serial
+import socket
+import sys
+import threading
+import readline
+import time
+import logging
+import os
+import argparse
+import re
+import codecs
+import platform
```

Comment:
isn't there some unofficial rule to order imports alphabetically?

Reply:
No, and I would be against it.

Reply:
If there would need an ordering rule, I would more go from the most generic to the most specific, like for C headers. But it can be subjective for some ones and would not enforce it.
I remember a package behave differently if 'readline' was included before it but don't remember which one.

I think, flake8 checks that standard library comes first, then external modules/packages and then the local package ones at the end but no hierarchy within the groups.